src/components/molecules/Button/styles.module.scss

```diff
@@ -100,14 +100,23 @@
   @include buttonTextStyle;
   padding: spacing(2) spacing(10);
 
+  &.customButtonPadding {
```

Comment:
**[blocker]**

Play through all button variants with and without icon with these changes. Currently sth seems to be wrong. For example the primary button we have as an example has wrong height, when it has an icon. But go over other buttons as well with all appearance + size + with/without icon combos to make sure we don't have any other similar cases